src/index.js

```diff
@@ -1,17 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
```

Comment:
- [ ] You don't have to delete the `App` component since this is the root component of our app. Kindly bring it back. ✍🏻  The `App.js` (main component) should have the imported `Calculator.js` component. don't import it directly from the index.js file. Your code should look like this:

```jsx
import Calculator from ./components/Calculator; // Import Calculator component

function App() {
  return (
    <Calculator />
  );
}
```